frontend/src/scenes/insights/insightDataLogic.tsx

```diff
@@ -256,12 +257,18 @@ export const insightDataLogic = kea<insightDataLogicType>([
             actions.insightLogicSaveInsight(redirectToViewMode)
         },
         saveAs: async () => {
-            promptLogic({ key: `save-as-insight` }).actions.prompt({
+            LemonDialog.openForm({
                 title: 'Save as new insight',
-                placeholder: 'Please enter the new name',
-                value: `${values.insight.name || values.insight.derived_name} (copy)`,
-                error: 'You must enter a name',
-                success: actions.saveAsNamingSuccess,
+                initialValues: { name: `${values.insight.name || values.insight.derived_name} (copy)` },
+                content: (
+                    <LemonField name="name">
```

Comment:
This should be `name="insight-name"` or something like that. With just "name", autocomplete (such as 1Password's) think this input refers to a person's name.

I see now there's a number of `Field name="name"` cases in our codebase, we should resolve these too at some point